news-frontend/src/components/SearchResults/SearchResults.js

```diff
@@ -1,13 +1,21 @@
 import React from "react";
 import "./SearchResults.css";
 import NewsCardList from "../NewsCardList/NewsCardList";
+import { useArticles } from "../../contexts/ArticleContext";
 
 const SearchResults = () => {
+  const { setToShowMore } = useArticles();
+
   return (
     <div className="search-results">
       <h2 className="search-results__title">Search results</h2>
       <NewsCardList />
-      <button className="search-results__button">Show more</button>
+      <button
+        className="search-results__button"
+        onClick={() => setToShowMore((prev) => (prev += 3))}
```

Comment:
`        onClick={() => setToShowMore((prev) => (prev += 3))}
`

the criteria say: Arrow functions are not passed as handler functions.


```jsx

// should be defined here
  const addMore = () => setToShowMore((prev) => (prev += 3))

  return (
    <div className="search-results">
      <h2 className="search-results__title">Search results</h2>
      <NewsCardList />
      <button
        className="search-results__button"
// used like that here
        onClick={addMore}

```

Reply:
thanks for the comment, done.